### ISSUE TITLE
Many to Many interface query broken (on item updates) [INTERNAL_SERVER_ERROR] Operator _none not supported

### ISSUE THREAD

**Issue by Dominic-Marcelino** [directus/directus#13435]
### Describe the Bug

After upgrading to 9.11.0 the many to many interface seems to be broken.
We can't display any existing relations on an existing item anymore as it generates an invalid query

The failing api call:
```
http://0.0.0.0:8055/items/relation?limit=25&fields[]=another_field&fields[]=title&fields[]=id&sort[]=id&page=1&filter=%7B%22_and%22:[%7B%22$FOLLOW(collection_relation,relation_id)%22:%7B%22_none%22:%7B%22collection_id%22:%7B%22_eq%22:%221%22%7D%7D%7D%7D]%7D&meta[]=filter_count&meta[]=total_count
```

### To Reproduce

* create two collections: "collection" and "relational"
* Add random field(s) to "relational"
* Add a many to many field to "collection"

* Add some fields to "relational"
* Create an item in "collection" (in this step the relational interface query still works)
* Save the item
* try to edit it and select other relations --> query fails

### Errors Shown

[INTERNAL_SERVER_ERROR] Operator _none not supported

### What version of Directus are you using?

9.11

### What version of Node.js are you using?

16.4

### What database are you using?

MySql

### What browser are you using?

Chrome

### How are you deploying Directus?

locally

**Comment by azrikahar** [comment]
Seems to be due to `_none` and `_some` was removed in #11737 ([specific code block](https://github.com/directus/directus/pull/11737/files#diff-5885d1acf50bd00ba4d0423d591bc471c868e92766b253bfa1e9bd768bf58c26L386-L389)), but it is still being used in files, list-m2a and list-m2m interfaces. There's also documentation on these 2 operators which should also be removed.

**Comment by u12206050** [comment]
Yeh also getting this problem, and since we do make use of a lot of m2m relationships this is causing yet again a rollback all the way back to Directus 9.8.0 :( Unless a hotfix could be pushed for this? @rijkvanzanten

**Comment by koksikus** [comment]
> Yeh also getting this problem, and since we do make use of a lot of m2m relationships this is causing yet again a rollback all the way back to Directus 9.8.0 :( Unless a hotfix could be pushed for this? @rijkvanzanten
You're right, I also think that 9.11.1 should be released quickly to fix this bug and not go backwards with the installation...

**Comment by fballan93** [comment]
I'm having this issue too . i thought I was going crazy until I saw my logs and it matches the current issue reported

**Comment by Dominic-Marcelino** [comment]
Any chance this can be released soon as a micro update?
Otherwise we’ve probably go back to v9.8 (without the relational rework) as the other versions also had some bugs in relational interfaces 😅

**Comment by battosai30** [comment]
Same here :( 
v9.11

**Comment by sergi-rourera-housfy** [comment]
We are having this issue too. Any ETA?

v9.11 too

**Comment by nickrum** [comment]
@sergi-rourera-housfy This should've been fixed in `v9.11.1`.

**Comment by battosai30** [comment]
I confirm : after upgrade to v9.11.1 it's solved for me ;)

Thanks for the quick fix !
